### PR TITLE
feat: [CDS-89723]: Skip build if it contains keywords like skip ci,ci skip,skip pipeline,pipeline skip,no_ci 

### PR DESCRIPTION
**Pull request by krishi0408**
Thanks for contributing to the Harness Developer Hub! Our code owners will review your submission.

## Description

* Please describe your changes: \__________________________________
* Jira/GitHub Issue numbers (if any): \______________________________
* Preview links/images (Internal contributors only): \__________________

## PR lifecycle

We aim to merge PRs within one week or less, but delays happen sometimes.

If your PR is open longer than two weeks without any human activity, please tag a [code owner](https://github.com/harness/developer-hub/blob/main/.github/CODEOWNERS) in a comment.

PRs must meet these requirements to be merged:

- [ ] Successful preview build.
- [ ] Code owner review.
- [ ] No merge conflicts.
- [ ] Release notes/new features docs: Feature/version released to at least one prod environment.

### CODE REVIEW

File: docs/platform/triggers/triggering-pipelines.md [hunk 1]
```diff
@@ -460,4 +461,40 @@ Leave out by the `synchronize` and `close` events from the event selection. Thes
...
+- [ci skip]
+- [skip pipeline]
+- [pipeline skip]
+- [no_ci]
+
+### Text Position
+
+The keywords must appear anywhere within the commit message, even if surrounded by other text. For example, This is a [ci skip] commit.
+
+### Special Characters
```
Comment (ayushitiwari024) [inline docs/platform/triggers/triggering-pipelines.md]: Need to remove this documentation. For special characters it won't work. We are only bound to use [skip ci], [ci skip], [pipeline skip], [skip pipeline], [no_ci]

Comment (krishi0408) [reply docs/platform/triggers/triggering-pipelines.md]: done removed

**Review by ayushitiwari024 — DISMISSED** [review]

**Review by ayushitiwari024 — DISMISSED** [review]
Lgtm

File: docs/platform/triggers/triggering-pipelines.md [hunk 2]
```diff
@@ -460,4 +461,38 @@ Leave out by the `synchronize` and `close` events from the event selection. Thes
...
+
+The keywords must appear anywhere within the commit message, even if surrounded by other text. For example, This is a [ci skip] commit.
+
+### Supported Platforms
+
+This functionality is applicable across various version control platforms, including:
+
+- GitHub
+- GitLab
+- Bitbucket
```
Comment (vcalasansh) [inline docs/platform/triggers/triggering-pipelines.md:495]: Should we add a blank line here?

File: docs/platform/triggers/triggering-pipelines.md [hunk 3]
```diff
@@ -460,4 +461,38 @@ Leave out by the `synchronize` and `close` events from the event selection. Thes
...
+
+- [skip ci]
+- [ci skip]
+- [skip pipeline]
+- [pipeline skip]
+- [no_ci]
+
+### Text Position
+
+The keywords must appear anywhere within the commit message, even if surrounded by other text. For example, This is a [ci skip] commit.
```
Comment (vcalasansh) [inline docs/platform/triggers/triggering-pipelines.md:487]: Enclose the example in quotes or some different formatting from main text.

Comment (krishi0408) [reply docs/platform/triggers/triggering-pipelines.md:487]: sure done

File: docs/platform/triggers/triggering-pipelines.md [hunk 4]
```diff
@@ -460,4 +461,38 @@ Leave out by the `synchronize` and `close` events from the event selection. Thes
...
 If you receive this error, it likely means you have not integrated [OAuth with Git Experience](/docs/platform/git-experience/oauth-integration) in Harness.
-For more information, go to [Harness Git Experience Overview](/docs/platform/git-experience/git-experience-overview).
\ No newline at end of file
+For more information, go to [Harness Git Experience Overview](/docs/platform/git-experience/git-experience-overview).
+
+
+## Skipping CI/CD Pipeline Builds
+:::note
+
+Currently, the following service dashboard functionality is behind the feature flag `CDS_SKIP_WEBHOOK_TRIGGER_EXECUTION_ON_SPECIAL_KEYWORDS`. Contact [Harness Support](mailto:support@harness.io) to enable the feature.
```
[the remark this answers is not in the document]
Comment (krishi0408) [reply docs/platform/triggers/triggering-pipelines.md:470]: thanks for pointing out, removed it from there

**Pull request merged**